src/lib/components/Tree.vue

```diff
@@ -12,16 +12,18 @@
         :gap="gap"
         :expand-row-by-default="reactiveExpandRowByDefault"
         :row-hover-background="rowHoverBackground"
+        :indeterminate-status="isIndeterminate.status"
         @emitNodeExpanded="onNodeExpanded"
         @emitOnUpdated="onDataUpdated"
         @emitCheckboxToggle="onCheckboxToggle"
       >
-        <template #checkbox="{ toggleCheckbox, checked }">
+        <template #checkbox="{ toggleCheckbox, checked, indeterminateStatus }">
```

Comment:
Is there a reason why we give it as `indeterminateStatus` instead of `indeterminate` directly?

Reply:
I made such a choice as a name. I'm editing it to make it more readable.